app/main.py

```diff
@@ -1,8 +1,31 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
+        self.wife = None
+        self.husband = None
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = []
```

Comment:
it's better to create first all instances of this class and than  assign the wife and husband attributes to the relevant person